README.md

```diff
@@ -80,6 +80,14 @@ Options
         Allows you to use your own reflector
         Default: http://reflector2.datastax.com/reflector2.php
 
+    --repository <repository>:<branch>
```

Comment:
I think this might need to be 2 options.  A repo can look like a wide variety of things including several URL formats and filesystem paths (GIT_URL and REMOTES sections of https://www.kernel.org/pub/software/scm/git/docs/git-fetch.html). Are we sure there's a character that's safe to split on? Colon is certainly not safe (shows up in all the URL formats). I'd say leave the repo by itself in this option.

I think we can use the existing --force_commit for the branch-reference.